docs/src/pages/components/data-grid/style/style.md

```diff
@@ -98,6 +98,14 @@ const columns: GridColumns = [
 
 {{"demo": "pages/components/data-grid/style/StylingCellsGrid.js", "bg": "inline"}}
 
+## Using the `classes` prop
+
+When the `className` prop isn't enough, and you need to access deeper elements, you can take advantage of the `classes` prop to customize the CSS injected by Material-UI.
+
+{{"demo": "pages/components/data-grid/style/StylingWithClassesProp.js", "bg": "inline"}}
```

Comment:
@oliviertassinari How about we remove the demo but keep the section with a link to the detailed explanation?

```suggestion

```

Reply:
We do no longer document the `classes` API in the customization guide in v5. IMHO, the problem we are going after here should be defered to post stable release. I

Reply:
So just remove the entire section.

Reply:
Removed